backend/substrapp/metrics.py

```diff
@@ -0,0 +1,62 @@
+from django.conf import settings
+from statsd import defaults, StatsClient
+from functools import wraps
+
+
+class NullStatsClient(StatsClient):
+    def timing(self, *args, **kwargs):
+        pass
+
+    def incr(self, *args, **kwargs):
+        pass
+
+    def decr(self, *args, **kwargs):
+        pass
+
+    def gauge(self, *args, **kwargs):
+        pass
+
+    def set(self, *args, **kwargs):
+        pass
+
+    def timer(self, *args, **kwargs):
+        def passthrough(f):
+            @wraps(f)
+            def _wrapped(*fargs, **fkwargs):
+                return f(*fargs, **fkwargs)
+
+            return _wrapped
+        return passthrough
+
+
+_statsd_client = None
```

Comment:
I think it's better to use a Class with class methods to do this stuff no ?

Reply:
Sorry I don't get the point here, could you provide more information please